src/Locales/LocaleContext.jsx

```diff
@@ -1,33 +1,30 @@
 /* global require */
 import React from 'react';
 import { ConfigProvider } from 'antd';
-import { emptyFn } from '.';
+import { emptyFn } from '..';
 
 export const DEFAULT_LOCALE = 'en_US';
 
 export const LocaleContext = React.createContext({
     locale: DEFAULT_LOCALE,
-    setLocale: emptyFn,
+    setLocale: emptyFn
 });
 
 export const LocaleProvider = ({ locale, setLocale, children }) => {
     let l10n;
 
     try {
-        let antdL10n = require(`antd/lib/locale/${locale}.js`).default;
-        let localL10n = require(`./Locales/${locale}.js`);
-        l10n = { ...antdL10n, ...localL10n };
+        l10n = require(`antd/lib/locale/${locale}.js`).default;
+        require.resolve(`./${locale}.js`);
```

Comment:
ja, das ist nur ein test ob es die übersetzungsdatei gibt und dann ggf. in den fallback zu gehen

Reply:
achso, verstehe

Reply:
ich verstehe nicht ganz was hier passiert. require.resolve gibt nur den filename zurück oder?